resources/views/checkout/partials/sidebar/totals.blade.php

```diff
@@ -1,26 +1,27 @@
 <div>
     <dt>@lang('Subtotal')</dt>
-    <dd>@{{ cart.prices.subtotal_including_tax.value | price }}</dd>
+    <dd v-cloak>@{{ cart.prices.subtotal_including_tax.value | price }}</dd>
 </div>
 <div v-if="cart.prices.applied_taxes.length">
```

Comment:
Here https://github.com/rapidez/checkout-theme/pull/126/files#r1827395330 may not apply since we don't know whether there are taxes applied, so https://github.com/rapidez/checkout-theme/pull/126/files#r1827374406 would suffice